Bookish/Bookish.Web/Views/Home/AddedBook.cshtml

```diff
@@ -0,0 +1,12 @@
+﻿@model Bookish.Web.Models.AddedBookViewModel
+@{
+    ViewData["Title"] = "AddedBook";
+}
+
+<h1>@Model.Title has been successfully added! Here are your barcodes</h1>
```

Comment:
I think "here are your barcodes" would be more suitable as a subheader as this header is a bit long 🙂